chia/_tests/blockchain/blockchain_test_utils.py

```diff
@@ -10,19 +10,20 @@
 from chia.consensus.blockchain import AddBlockResult, Blockchain
 from chia.consensus.difficulty_adjustment import get_next_sub_slot_iters_and_difficulty
 from chia.consensus.multiprocess_validation import PreValidationResult, pre_validate_block
+from chia.full_node.block_store import BlockStore
 from chia.types.validation_state import ValidationState
 from chia.util.errors import Err
 
 
 async def check_block_store_invariant(bc: Blockchain):
-    db_wrapper = bc.block_store.db_wrapper
+    block_store = cast(BlockStore, bc.block_store)
```

Comment:
This is ugly for sure, but we can get away with it because we know that `bc` has been populated with a `BlockStore` instance here.

In future PRs, I expect I will have to rework some tests pretty significantly.